File: TestVBAxis2Bitmap/TestVBAxis2Bitmap/NoScrollPanel.vb
Option Strict On
Option Explicit On

Friend Class NoScrollPanel
    Inherits Panel

    Protected Overrides Sub OnMouseWheel(ByVal e As MouseEventArgs)
        Dim mouseEvent As HandledMouseEventArgs = DirectCast(e, HandledMouseEventArgs)
        mouseEvent.Handled = True
    End Sub
End Class

File: SEGYReader/NoScrollPanel.cs
using System.Windows.Forms;

namespace SegyView
{
    public class NoScrollPanel : Panel
    {
        protected override void OnMouseWheel(MouseEventArgs e)
	    {
		    var mouseEvent = (HandledMouseEventArgs)e;
		    mouseEvent.Handled = true;
	    }
    }
   
}
